iaso/api/form_attachments.py

```diff
@@ -41,16 +43,19 @@ def validate(self, data: typing.MutableMapping):
 
     def create(self, validated_data):
         form: Form = validated_data["form"]
-        file: File = validated_data["file"]
+        file: InMemoryUploadedFile = validated_data["file"]
```

Comment:
in term of types, I'm not you'll always get there are several implementation : InMemoryUploadedFile, TemporaryUploadedFile and depends on the django settings.

may be it's better to use or a more io type.
```
from django.core.files.uploadedfile import UploadedFile
```